handlers/kinetic_request_ce_submission_db_bulk_upsert_v1/handler/init.rb

```diff
@@ -190,6 +190,17 @@ def initialize(input)
 #
 #######################################################################################################
   def execute
+    # Get kapp fields and add them to the kapp_fields list
+    @@KAPP_FIELDS = get_kapp_fields({
```

Comment:
I was attempting to make as few changes as possible.  The class variable was there and originally written by Ben, I think.  Because the handler does threading, I'm concerned about changing it from a class variable.

Reply:
OK